app/package.json

```diff
@@ -27,8 +27,8 @@
     "@types/react-dom": "^18.0.5",
     "axios": "^0.27.2",
     "date-fns": "^2.28.0",
-    "devextreme": "21.2.7",
-    "devextreme-react": "21.2.7",
+    "devextreme": "^22.1.3",
+    "devextreme-react": "^22.1.3",

```

Comment:
Probably not a great idea to be upgrading our packages this late in the project, let's revert to original devextreme, devextreme-react, and universal-cookie packages so we don't encounter any gotchas or breaking changes.